listener/pkg/event/skr_events_listener.go

```diff
@@ -22,11 +21,6 @@ const (
 	requestSizeLimitInBytes = 16384 // 16KB
 )
 
-func RegisterListenerComponent(addr, componentName string, verify Verify) (*SKREventListener, *source.Channel) {
```

Comment:
Note to the reviewer: This function is obsolete because of the changes necessary in the corresponding PR in the `lifecycle-manager` project.